runtime/android/core_internal/src/org/xwalk/core/internal/XWalkSettingsInternal.java

```diff
@@ -784,8 +786,12 @@ public void run() {
     }
 
     /**
-     * See {@link android.webkit.WebSettings#getSaveFormData}.
+     * Gets whether the XWalkView saves form data.
+     * @return whether the WebView saves form data
+     * @see #setSaveFormData
```

Comment:
Understand the different between them:
{@link} is an inline link and can be placed wherever you like
@see creates its own section

That means @link is part of your description, like the most link here: https://crosswalk-project.org/apis/embeddingapidocs_v6/org/xwalk/core/package-summary.html, while the result of @see as below:

> Since:
7.0
See Also:
setSaveFormData(boolean)